### PR TITLE
Touch Bar and suspend work

### PR DESCRIPTION
**Pull request by AdityaGarg8**

### CODE REVIEW

File: docs/state.md [hunk 1]
```diff
@@ -9,15 +9,15 @@ While Linux is usable on all T2 models, some features are limited due to the lac
...
+- Touch Bar
+- Suspend (It works if [this guide](https://wiki.t2linux.org/guides/dkms/#fixing-suspend) is followed.
 
 ## Partially Working
 
 - Bluetooth: Not working on models with the BRCM4377 WiFi chip (MacBookPro15,4, MacBookPro16,3, MacBookAir9,1), and models with the BRCM4355 WiFi chip (MacBookAir8,1, MacBookAir8,2).
 - Trackpad: Though it is technically working, it is far from the experience on macOS. No force touch or palm rejection. Some models have deadzones on the edges of their trackpads.
-- Touchbar: There is support for the so called simple mode, the same that you would see on Bootcamp Windows for example. Either function keys from 1 to 12 or basic media / brightness control are shown. Sometimes it is unable to change between function keys and media / brightness keys.
 - Audio: With proper configuration audio can work, however it is not stable in some older kernels and switching between speakers and when using the microphone. Microphone volume is low in some Macs.
-- Suspend is very slow to resume (20-40 seconds), and the Touchbar sometimes does not work after resume.
```
Comment (Redecorating) [inline docs/state.md:20]: Can we keep the "Very slow to resume bit"? It shouldn't be nearly as slow as it is (iirc the thunderbolt/usb-c ports were taking too long according to https://github.com/intel/pm-graph )

File: docs/state.md [hunk 2]
```diff
@@ -9,15 +9,15 @@ While Linux is usable on all T2 models, some features are limited due to the lac
 - USB
 - Keyboard
 - Camera
-- Wifi (For now its working for all the devices we have data about. There are still some devices of whom we don't have data. Refer to the [wifi guide](https://wiki.t2linux.org/guides/wifi/) for more details)
+- Wi-Fi
+- Touch Bar
```
Comment (Redecorating) [inline docs/state.md:13]: Maybe still mention that it's the same as the windows driver, no custom graphics.

**Review by Redecorating — CHANGES_REQUESTED** [review]

File: docs/state.md (hunk 2 above)
Comment (AdityaGarg8) [reply docs/state.md]: 4d7e402

File: docs/state.md (hunk 1 above)
Comment (AdityaGarg8) [reply docs/state.md:20]: 4d7e402

Comment (Redecorating) [reply docs/state.md:20]: Could you also give a number of seconds?

Comment (AdityaGarg8) [reply docs/state.md:20]: How much?
Takes around 5-10 for me since the touchbar got fixed

Comment (Redecorating) [reply docs/state.md:20]: I haven't tried suspend since that was fixed, I'll go check if that has improved it

Comment (Redecorating) [reply docs/state.md:20]: with one test i got 16 seconds

Comment (AdityaGarg8) [reply docs/state.md:20]: I got 9 rn

Comment (Redecorating) [reply docs/state.md:20]: looks like it's not as slow then!

> its too slow

maybe "it's very slow (5-20 seconds)". "Too" makes me think this slowness breaks something, but it's just slow.

Comment (AdityaGarg8) [reply docs/state.md:20]: ed3ce16

**Review by Redecorating — APPROVED** [review]

**Pull request merged**